_data/simple-icons.json

```
@@ -3639,6 +3644,6 @@
             "title": "Zulip",
             "hex": "52C2AF",
             "source": "https://github.com/zulip/zulip/"
-        }
+        }        
     ]
-}
+}
```

Just to not litter the commit history, can you revert the change here (i.e. add a final newline to the file)?

```suggestion
}

```